### ISSUE TITLE
New component: Active Directory Domain Services Receiver

### ISSUE THREAD

**Issue by BinaryFissionGames** [open-telemetry/opentelemetry-collector-contrib#8833]
## The purpose and use-cases of the new component
The Active Directory Domain Services Receiver would collect performance metrics relating to an [Active Directory Domain Services](https://docs.microsoft.com/en-us/windows-server/identity/ad-ds/get-started/virtual-dc/active-directory-domain-services-overview) instance. 

Similar to the proposed [IIS](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/8370) and [MSSQL](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/8398) receivers, this receiver would leverage the existing `windowsperfcountersreceiver` to collect performance counter metrics.

Targeted versions are the Active Directory versions included with the following versions of Windows:
* Windows Server 2012 R2
* Windows Server 2016
* Windows Server 2019
* Windows Server 2022

## Example configuration for the component
```yaml
receivers:
  activedirectoryds:
    collection_interval: 5s
```

## Telemetry data types supported
Metrics

**Comment by djaglowski** [comment]
I'll sponsor this.

**Comment by djaglowski** [comment]
Closed by #9359
